### PR TITLE
PR5720-Follow-up Minor Edits

### PR DESCRIPTION
**Pull request by bmcelvee**
Follow-up to https://github.com/openshift/openshift-docs/pull/5720.

### CODE REVIEW

**Comment by bmcelvee** [comment]
@coreydaley PTAL. I want to make sure nothing was lost in the edits 😸

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 1]
```diff
@@ -12,51 +12,62 @@
...
-  // select the project that you want to use
-  // or create a new project with oc new-project <project_name>
-  $ oc project <project_name>
-  // if you want to use persistent storage, use jenkins-persistent instead
-  $ oc new-app jenkins-ephemeral
+  $ oc project <project_name> <1>
+  $ oc new-app jenkins-ephemeral <2>
 ----
+<1> Select the project that you want to use or create a new project with `oc`
+new-project <project_name>.
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: `oc new-project <project_name>`

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 2]
```diff
@@ -12,51 +12,62 @@
...
+<2> If you want to use persistent storage, use `jenkins-persistent` instead.
 
 [NOTE]
 ====
-If Jenkins auto-provisioning is enabled on your cluster, and you don't need to make any customizations to the
-Jenkins master, you can skip the above step. For more information about Jenkins autoprovisioning see xref:../../install_config/configuring_pipeline_execution.html[Configuring Pipeline Execution].
+If Jenkins auto-provisioning is enabled on your cluster, and you don't need to
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: s/don't/do not

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 3]
```diff
@@ -69,63 +80,62 @@ spec:
     type: JenkinsPipeline
 ----
 
-For more information about configuring the Pipeline Build Strategy see xref:../../builds/build_strategies.html#pipeline-strategy-options[Pipeline Strategy Options].
+For more information about configuring the Pipeline Build Strategy see
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: Strategy, see

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 4]
```diff
@@ -69,63 +80,62 @@ spec:
...
+xref:../builds/build_strategies.adoc#pipeline-strategy-options[Pipeline
+Strategy Options].
 
 [[the-jenkinsfile]]
-
 == The Jenkinsfile
 
-Now that we have created the build config with a jenkinsPipelineStrategy, we need to tell the pipeline what to do.
-We will do this by using an inline Jenkinsfile, since we aren't going to be creating our own git repository for this application.
+Once you have created a BuildConfig with a `jenkinsPipelineStrategy`, tell the
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: Once you create

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 5]
```diff
@@ -69,63 +80,62 @@ spec:
...
+pipeline what to do by using an inline `jenkinsfile`. This example does not set
+up a git repository for the application.
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: Git repository

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 6]
```diff
@@ -69,63 +80,62 @@ spec:
...
 
-Below is the Jenkinsfile content which is written in groovy using the OpenShift DSL.  For this example we will inline
-this content into the build config using the http://www.yaml.org/spec/1.2/spec.html#id2795688[YAML Literal Style], though
-including a Jenkinsfile in your source repository is the preferred Jenkins way of doing things.
+The following `jenkinsfile` content is written in groovy using the OpenShift
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: Groovy

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 7]
```diff
@@ -145,79 +154,101 @@ openshift.withCluster() {
...
+<9> Wait up to five minutes for the deployment to complete.
+<10> If everything else succeeded, tag the `$ {templateName}:latest` image as
+`$ {templateName}-staging:latest`. A pipeline BuildConfig for the staging
+environment can watch for the `$ {templateName}-staging:latest` image to change
+and then deploy it to the staging environment.
 
 [NOTE]
 ====
-The above example has been written using the *declarative pipeline* style, but the older *scripted pipeline* style is also supported.
+The previous example has been written using the *declarative pipeline* style,
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: was written using

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 8]
```diff
@@ -145,79 +154,101 @@ openshift.withCluster() {
...
 
 [source]
+
 ----
 $ oc create -f nodejs-sample-pipeline.yaml
 ----
 
-If you don't want to create your own file, you can use the sample from the Origin repository with the following command:
+If you don't want to create your own file, you can use the sample from the
+Origin repository by running:
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: s/don't/do not

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 9]
```diff
@@ -145,79 +154,101 @@ openshift.withCluster() {
...
- * the pipeline runs on the slave pod _(or the master if no slave is required)_
- ** any previously created resources with the *template=nodejs-mongodb-example* label will be deleted
- ** a new application (and all of it's associated resources) will be created from the *nodejs-mongodb-example* template
- ** a build will be started using the *nodejs-mongodb-example* build config
- *** the pipeline will wait until the build has completed to trigger the next stage
- ** a deployment will be started using the *nodejs-mongodb-example* deployment config
- *** the pipeline will wait until the deployment has completed to trigger the next stage
- ** if the build and deploy are successful, the *nodejs-mongodb-example:latest* image will be tagged as *nodejs-mongodb-example:stage*
- * the slave pod is deleted _(if one was required for the pipeline)_
+Once the pipeline has started you should see the following actions performed
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: is started, you should

File: dev_guide/app_tutorials/openshift_pipeline.adoc [hunk 10]
```diff
@@ -145,79 +154,101 @@ openshift.withCluster() {
...
+within your project:
+
+* A job instance is created on the Jenkins server.
+* A slave pod is launched, if your pipeline requires one.
+* The pipeline runs on the slave pod, or the master if no slave is required.
+** Any previously created resources with the `template=nodejs-mongodb-example`
+label will be deleted.
+** A new application, and all of it's associated resources, will be created from
```
Comment (ahardin-rh) [inline dev_guide/app_tutorials/openshift_pipeline.adoc]: its associated resources

File: dev_guide/openshift_pipeline.adoc [hunk 11]
```diff
@@ -11,105 +11,133 @@
...
 [[jenkins-pipeline-strategy]]
-
 == Jenkins Pipeline Strategy
 
-In order to take advantage of the OpenShift Pipelines within your project, you will need to utilize the
-xref:../../dev_guide/builds/build_strategies.html#pipeline-strategy-options[Jenkins Pipeline Build Strategy].
-This strategy defaults to using a *Jenkinsfile* at the root of your source repository, but also provides the
-following configuration options:
+In order to take advantage of the OpenShift Pipelines within your project, you
+will need to utilize the
```
Comment (ahardin-rh) [inline dev_guide/openshift_pipeline.adoc]: s/you will need to utilize/you must use

File: dev_guide/openshift_pipeline.adoc [hunk 12]
```diff
@@ -11,105 +11,133 @@
...
+The optional `jenkinsfilePath` field specifies the name of the file to use,
+relative to the source `contextDir`. If `contextDir` is omitted, it defaults to
+the root of the repository. If `jenkinsfilePath` is omitted, it defaults to
+`jenkinsfile`.
 ====
 
-For more detailed information about the Jenkins Pipeline Strategy see xref:../../dev_guide/builds/build_strategies.html#pipeline-strategy-options[Pipeline Strategy Options].
-
-[[the-jenkinsfile]]
+For more detailed information about the Jenkins Pipeline Strategy see
```
Comment (ahardin-rh) [inline dev_guide/openshift_pipeline.adoc]: , see

File: dev_guide/openshift_pipeline.adoc [hunk 13]
```diff
@@ -11,105 +11,133 @@
...
+OpenShift Client binaries available if you intend to use the OpenShift DSL.
 
+[[pipeline-tutorial]]
 == Tutorial
 
-For a full walkthrough of building and deploying an application via a Jenkins Pipeline see xref:../app_tutorials/jenkins_pipeline.html#overview[Jenkins Pipeline Tutorial].
-
-[[advanced-topics]]
+For a full walkthrough of building and deploying an application with Jenkins
+Pipeline see xref:../dev_guide/app_tutorials/openshift_pipeline.adoc#overview[Jenkins
```
Comment (ahardin-rh) [inline dev_guide/openshift_pipeline.adoc]: , see

File: dev_guide/openshift_pipeline.adoc [hunk 14]
```diff
@@ -11,105 +11,133 @@
...
+The link:https://wiki.jenkins.io/display/JENKINS/Kubernetes+Plugin[Kubernetes
+Plug-in] is also pre-installed in the official Jenkins image. This plug-in
+allows the Jenkins master to create slave pods on OpenShift and delegate running
+jobs to them to achieve scalability as well as providing pods with specific
 runtimes for specific jobs.
 
-For more detailed information on configuring slave pods using the Kubernetes Plugin see https://github.com/jenkinsci/kubernetes-plugin/blob/master/README.md[Kubernetes Plugin README].
-
+For more detailed information on configuring slave pods using the Kubernetes
+Plug-in see
```
Comment (ahardin-rh) [inline dev_guide/openshift_pipeline.adoc]: , see

**Comment by coreydaley** [comment]
@bmcelvee lgtm

**Pull request merged**